e2e/fixtures/authenticatedUserPages.ts

```diff
@@ -100,7 +100,7 @@ export const testAuth = base.extend<
     },
     // specify timeout so worker fixture duration doesn't skew test duration
     // ...see https://github.com/microsoft/playwright/issues/15338
-    { scope: 'worker', timeout: 2 * 60 * 1000 },
+    { scope: 'worker', timeout: defaultTestTimeout * 2 },
```

Comment:
Fixtures intermittently timed out on linux runner -- allow time to create/setup two users